CHANGELOG.adoc

```diff
@@ -5,6 +5,70 @@ https://spring.io/projects/spring-cloud-gcp[Spring Cloud GCP] is a set of integr
 This document provides a high-level overview of the changes introduced in Spring Cloud GCP by release.
 For a detailed view of what has changed, refer to the https://github.com/spring-cloud/spring-cloud-gcp/commits/master[commit history] on GitHub.
 
+== 1.2.2.RELEASE (2020-03-04)
+
+=== General
+
+* Switched to using GCP Libraries BOM for managing GCP library versions (https://github.com/spring-cloud/spring-cloud-gcp/issues/2109[#2109])
+* Core auto-configuration can now be disabled with `spring.cloud.gcp.core.enabled=false` (https://github.com/spring-cloud/spring-cloud-gcp/issues/2147[#2147])
+* Reference documentation improvements
+* Two new modules: Firebase Auth and Secret Manager
+
+=== Datastore
+
+* Support lazy loading entities using @LazyReference (https://github.com/spring-cloud/spring-cloud-gcp/issues/2104[#2104])
+* Made existsById more efficient by retrieving only the key field (https://github.com/spring-cloud/spring-cloud-gcp/issues/2127[#2127])
+* Projections now work with the Slice return type (https://github.com/spring-cloud/spring-cloud-gcp/issues/2133[#2133]) and GQL queries (https://github.com/spring-cloud/spring-cloud-gcp/issues/2139[#2139]) in repositories
+* Improved repository method name validation (https://github.com/spring-cloud/spring-cloud-gcp/issues/2155[#2155])
+* Fixed delete for void repository method return type (https://github.com/spring-cloud/spring-cloud-gcp/issues/2169[#2169])
+
+=== Firebase (NEW)
+
+* Introduced Firebase Authentication module (https://github.com/spring-cloud/spring-cloud-gcp/issues/2111[#2111])
+
+=== Firestore
+
+* Added IN support in name-based queries (https://github.com/spring-cloud/spring-cloud-gcp/issues/2054[#2054])
+
+=== Pub/Sub
+
+* ACK_MODE is now configurable using stream binders (https://github.com/spring-cloud/spring-cloud-gcp/issues/2079[#2079])
+* Added HealthIndicator implementation (https://github.com/spring-cloud/spring-cloud-gcp/issues/2030[#2030])
+* Fixed: `PubSubReactiveFactory.poll` doesn't handle exceptions thrown by the `PubSubSubscriberOperations` (https://github.com/spring-cloud/spring-cloud-gcp/issues/2229[#2229])
```

Comment:
Add a "NOTE: previously silently ignored exceptions are now forwarded to the Flux"

Reply:
Done.